src/transformers/hf_api.py

```diff
@@ -219,17 +235,25 @@ def list_repos_objs(self, token: str, organization: Optional[str] = None) -> Lis
         d = r.json()
         return [RepoObj(**x) for x in d]
 
-    def create_repo(self, token: str, name: str, organization: Optional[str] = None) -> str:
+    def create_repo(
+        self, token: str, name: str, organization: Optional[str] = None, lfsmultipartthresh: Optional[int] = None
+    ) -> str:
         """
         HuggingFace git-based system, used for models.
 
         Call HF API to create a whole repo.
+
+        Params:
+            lfsmultipartthresh: Optional: internal param for testing purposes.
```

Comment:
can we change `lfsmultipartthresh` to `lfs_multi_part_thresh`? => very hard to read :D

Reply:
i'd rather keep the consistency with the server side on that one